_infra/helm/collection-instrument/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.1.36
+version: 2.1.37
```

Comment:
I think this could arguably be 2.2.0 or even 3.0.0 as theres no backwards compatibility after we do this change